### ISSUE TITLE
FR: Add long press -> open to Android app

### ISSUE THREAD

**Issue by johschmitz** [tailscale/tailscale#3644]
### What are you trying to do?

Open a webinterface of a device in my Tailscale network from the Tailscale Android app. 

### How should we solve this?

Let me long press the device and then "Open" in the web browser. Check out the "Network Analyzer" app for a good example how to implement this:
https://play.google.com/store/apps/details?id=net.techet.netanalyzerlite.an
Besically, you just need to let the user select the IP Adress as a text so that Android can display the context overlay menu.
![Screenshot_20211231-130356699](https://user-images.githubusercontent.com/771979/147822498-3259fdc8-13c5-459b-9a9f-4cd680f49552.png)

### What is the impact of not solving this?

It takes more steps to open a webinterface -> annoying UX. 

### Anything else?

While testing this I noticed that I can not use a network scanner app like "Ning" on the Tailscale network interface and discover devices like being connected to the physical home network. It just doesn't find the device that is shown in the Tailscale app. Might be a problem of the scanner app though. Didn't test with nmap so far.

**Comment by DentonGentry** [comment]
> While testing this I noticed that I can not use a network scanner app like "Ning" on the Tailscale network interface and discover devices like being connected to the physical home network. It just doesn't find the device that is shown in the Tailscale app. Might be a problem of the scanner app though.

Most such apps work in one of two ways (or both):
1. IP Broadcast, which doesn't work on Tailscale
1. Enumerating each IP address one-by-one, sending a ping to see if it is present.

On a typical LAN this enumberating IP addresses is something like a 192.168.1.0/24 address, 256 IP addresses to check. For Tailscale the IP address is added as a Point-to-Point link, 100.x.y.z/32, so the straightforward code to enumerate will decide there are no addresses other than that one. Even if the code tried, it would have to enumerate a 100.64.0.0/10 which is over a million IP addresses.

Network scanners don't really have a way to work on a wide-area VPN network like Tailscale, fr the same reason that you can't scan the entire Internet.

**Comment by johschmitz** [comment]
@DentonGentry Makes sense, once I saw the "second" saw the IP adresses and was suspecting you do something like that.
I still hope the original "convenience" feature request gets implemented somehow at some point.
